src/test/resources/com/stripe/model/charge_outcome.json

```diff
@@ -1,5 +1,6 @@
 {
     "network_status": "approved_by_network",
+    "risk_level": "normal",
```

Comment:
Sorry, miniscule nit, but given that this list is already alphabetized, can you move `risk_level` down a stop?